src/dataDisplay/icon/icon.jsx

```diff
@@ -36,13 +82,17 @@ const propTypes = {
      * Provides a human-readable title for the element that contains it.
      */
     title: TITLE_PROP_TYPE,
+    /**
+     * The string for the icon you want to display.
+     */
     type: PropTypes.string.isRequired,
 };
 
 const defaultProps = {
     align: 'left',
     className: null,
     color: 'primary',
+    dataTestId: `${UI_CLASS_NAME}-drawer`,
```

Comment:
Err, should be `-icon` rather than `-drawer`?

Reply:
Thanks

Reply:
Looks good.  Resolving.